tests/src/test/java/alluxio/master/file/CheckConsistencyIntegrationTest.java

```diff
@@ -52,6 +56,7 @@
 
   private FileSystemMaster mFileSystemMaster;
   private FileSystem mFileSystem;
+  private InodeTree mTree;
```

Comment:
Is this necessary? Could we just change the UFS owner and permissions instead?

Reply:
We can not change UFS owner because we don not know which owner can be used on UFS.